framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -587,6 +587,32 @@ public AnnotatedTypeFactory(BaseTypeChecker checker) {
             this.typeInformationPresenter = null;
         }
 
+        // Alias provided via -AaliasedTypeAnnos command-line option
+        if (checker.hasOption("aliasedTypeAnnos")) {
+            String aliasesOption = checker.getOption("aliasedTypeAnnos");
+            String[] annos = aliasesOption.split(";");
```

Comment:
At least on Unix systems, using ';' is a bit annoying, as it is the command separator. I can't really think of something better, but please add a warning with the explanation.

Another warning that would be good to add in the documentation: this can only be used for annotations without attributes, e.g. this will not be usable to declare an alias for `@Regex(2)`.
No need to implement this, just add a comment in the user manual and somewhere here.